lib/FieldType/Mapper/BinaryFileFormMapper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+class BinaryFileFormMapper implements FieldTypeFormMapperInterface
+{
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        $fieldDefinitionForm
+            ->add(
+                'maxSize',
+                'integer',
+                [
+                    'required' => true,
+                    'property_path' => 'validatorConfiguration[FileSizeValidator][maxFileSize]',
+                    'label' => 'field_definition.ezbinaryfile.max_file_size',
+                ]
+            );
```

Comment:
@lolautruche Interesting. PhpStorm says "Only one argument is allowed per line in a multi-line function call", though not if only the array declaration is multi-line. Ok, I'll do this.